Student-Class/src/student/pkgclass/StudentClass.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package student.pkgclass;
+
+/**
+ *
+ * @author Ziying Guo
+ */
+public class StudentClass {

```

Comment:
I name it studentclass because when I was creating the application I thought it's better if I name it as the assignment name.

Sent from my iPhone

> On May 7, 2015, at 11:35, Raffi Khatchadourian <notifications@github.com> wrote:
> 
> In Student-Class/src/student/pkgclass/StudentClass.java:
> 
> > @@ -0,0 +1,55 @@
> > +/*
> > + * To change this license header, choose License Headers in Project Properties.
> > + * To change this template file, choose Tools | Templates
> > + * and open the template in the editor.
> > + */
> > +
> > +package student.pkgclass;
> > +
> > +/**
> > + *
> > + * @author Ziying Guo
> > + */
> > +public class StudentClass {
> This isn't a student class. This is an application that uses the Student class. You can name this class something like StudentApplication. This class is just meant to have a main method.
> 
> —
> Reply to this email directly or view it on GitHub.
>